prime-router/src/main/kotlin/cli/tests/CompareData.kt

```diff
@@ -366,13 +366,19 @@ warnings: ${warnings.joinToString()}
 
 /**
  * Compares two HL7 files.
+ * @property result object to contain the result of the comparison
  */
-class CompareHl7Data(val result: CompareData.Result = CompareData.Result()) {
-    /**
-     * The list of fields that contain dynamic values that cannot be compared.  Source:
-     * Hl7Serializer.setLiterals()
-     */
-    internal val dynamicHl7Values = arrayOf("MSH-7", "SFT-2", "SFT-4", "SFT-6")
+class CompareHl7Data(
+    val result: CompareData.Result = CompareData.Result(),
+    private val ignoredFields: List<String> = covidDynamicHl7Fields
+) {
+    companion object {
+        /**
+         * The list of fields that contain dynamic values that cannot be compared.  Source:
+         * Hl7Serializer.setLiterals()
+         */
+        private val covidDynamicHl7Fields = listOf("MSH-7", "SFT-2", "SFT-4", "SFT-6")
```

Comment:
Are these fields dynamic for both COVID and non-covid data?

Reply:
Possibly, but until I go through the new HL7 I won't know.